lib/lhm/chunker.rb

```diff
@@ -55,12 +55,12 @@ def copy_batchwise(select_clause)
 
     def select_start
       start = connection.select_value("select min(#{origin_primary_key}) from #{ origin_name }")
-      start ? start.to_i : nil
+      start ? start.to_i : 1
     end
 
     def select_limit
       limit = connection.select_value("select max(#{origin_primary_key}) from #{ origin_name }")
-      limit ? limit.to_i : nil
+      limit ? limit.to_i : 1
```

Comment:
https://github.com/freshworks/large-hadron-migrator/blob/f451bd444d1866ecdab63eb037d69db0bf76c642/lib/lhm/chunker.rb#L97
Setting both start and limit to 1, will it not fail at validate step?

Reply:
No it won't fail, because this is similar to having 1 row in the table with the primary key as 1. Will add test cases for these.

Reply:
My bad, misread the condition. Thanks.

Reply:
@ImAjayS added the test cases, pleae check